framework/lib/components/date-picker/date-picker/date-range-picker.tsx

```diff
@@ -17,23 +18,43 @@ import { InputGroup, InputRightElement } from '../../input'
 import { Icon } from '../../icon'
 import { SimpleRangeCalendar } from '../components/calendar/simple-range-calendar'
 
+const parseValue = (value: { startDate: string, endDate: string } | undefined | string) => {
+  if (isNil(value) || typeof value === 'string') return value
```

Comment:
You need to add an  if `isValidDateRange(value)` parseDate(value) else return null check here. Otherwise if the value is not valid it would crash the app. Null signifies empty value. You can do this as an early return instead of the one you made to check if isNil / isString

Reply:
Fixed!